portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tony Shen - Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <!--Google Font API-->
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto&effect=anaglyph">
+        <script src="script.js"></script>
+    </head>
+    <body style="page-container">
+        <div class="axis-0-grid-container">
+            <div class="header-container">
+
+                <div class="logo-section">
+                    <img class="logo" src="images/profile.jpg"/>
+                </div>
+
+                <div class="title-section"><a class="font-effect-anaglyph title" href="index.html">Tony Shen</a></div>
+
+                <div class="navigator-section">  
+                    <a class="font-effect-anaglyph header-links" href="projects.html">Projects</a>
+                    <a class="font-effect-anaglyph header-links" href="https://drive.google.com/file/d/1lSsqU0wnAMAjw4Yj3lWPOef8z43ro9un/view?usp=sharing">CV</a>
+                    <a class="font-effect-anaglyph header-links" href="blog.html">Vlog</a>
+                    <a class="font-effect-anaglyph header-links" href="gallery.html">Images</a>
+                </div>
+
+            </div>
+
+            <div class="image-container">
+
+                <div class="image-column">
+                    <div>img</div>
```

Comment:
Please remove, or replace with a single instance of a valid image. Submitting test code is risky... once it is in the repo there is much higher risk it will find its way to production. For this reason we prefer to remove the test code before submission. You can keep it in a separate file, or copy back from the branch history if needed.

Reply:
Thanks for the heads up, I will go remove them